src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -95,6 +99,28 @@ public static boolean shouldBeHearingsInWelshFlag() {
 
     public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
                                                             ReferenceDataServiceHolder referenceDataServiceHolder) {
+        List<HearingLocation> hearingLocations = getAllHearingLocations(caseData, referenceDataServiceHolder);
+
+        String nextHearingVenueName = caseData.getAdjournCaseNextHearingVenue();
+
+        if (!referenceDataServiceHolder.isAdjournmentFlagEnabled() && isNotEmpty(nextHearingVenueName)
```

Comment:
Shouldn't this be no "!" ? If isAdjournmentFlagEnabled is false (like right now) then this whole block will be skipped.

Reply:
Done